### PR TITLE
Update README with Node.js 18

### PR DESCRIPTION
**Pull request by alanseiden**
Isn't Node.js 18 supported?

### CODE REVIEW

**Comment by kadler** [comment]
This change is so short that the review box gets cut off, lol

image

Works on Chrome, ie. the new IE, though.

**Review by kadler — APPROVED** [review]
LGTM. @markdirish should we remove Node 12 and Node 15? Both are out of support and 15 is out of place since it's not an LTS release.

**Comment by markdirish** [comment]
> LGTM. @markdirish should we remove Node 12 and Node 15? Both are out of support and 15 is out of place since it's not an LTS release.

I'm not sure why 15 is on there, probably a slip of the key. The supported versions should be 14, 16, and 18, with 14 going out of support on April 30th (and Node.js 20 coming out April 18th)

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ This package is a native addon, built with C++ code using `node-addon-api`, a C+
 * Node.js 14
 * Node.js 15
```
Comment (markdirish) [inline README.md:30]: Alan, could you remove Node.js 16 as well? Just to reduce the number of commits PRs, makes most sense to do it here.

**Review by markdirish — CHANGES_REQUESTED** [review]
Remove Node.js 15 too and we are good to go!

File: README.md (hunk 1 above)
Comment (alanseiden) [reply README.md]: @markdirish I've added another commit to the branch to remove version 15. Let me know if I need to rebase, but perhaps you can squash and merge.

Comment (markdirish) [reply README.md]: Looks good. Could you now make the DCO bot happy? There are instructions in this link: https://github.com/markdirish/node-odbc/pull/315/checks?check_run_id=12195617013

I believe it should be:

1. Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
2.  In your local branch, run: `git rebase HEAD~2 --signoff`
3. Force push your changes to overwrite the branch: `git push --force-with-lease origin patch-1`

**Comment by alanseiden** [comment]
@markdirish This issue was just marked stale but I think I did everything necessary on my end. Thanks!

**Review by markdirish — APPROVED** [review]
LGTM, Thanks Alan!

**Pull request merged**